common/config/base.php

```diff
@@ -147,7 +147,12 @@
                 'baseUrl' => env('STORAGE_BASE_URL'),
             ],
             require(Yii::getAlias('@storage/config/_urlManager.php'))
-        )
+        ),
+
+        'queue' => [
+            'class' => \yii\queue\file\Queue::class,
+            'path' => '@runtime/queue',
```

Comment:
pls, change to
```
@common/runtime/queue
```

``@runtime`` is relative to current application eg frontend, backend, console